scripts/zones/Northern_San_dOria/npcs/Helaku.lua

```diff
@@ -25,46 +25,44 @@ end;
 -----------------------------------
 
 function onTrigger(player,npc)
-	
-	currentMission = player:getCurrentMission(BASTOK);
-	
-	if(currentMission == THE_EMISSARY) then
+	local currentMission = player:getCurrentMission(BASTOK);
+	local missionStatus = player:getVar("MissionStatus");
+
+	if (currentMission == THE_EMISSARY) then
 		missionStatus = player:getVar("MissionStatus");

```

Comment:
My bad, I was intending to local an un-local'd var and missed that it was being defined already